src/Input.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+/**
+ * 테스트 목록
+ *
+ * Input에
+ *  value가 없을때
+ *    placehoder에 "할 일을 입력해 주세요" 가 노출되는지
+ *  value가 변경되었을때
+ *    onChange 함수가 실행되는지
+ *  button을 클릭했을때
+ *   onClick 함수가 실행되는지
+ */
+describe('Input', () => {
+  const handleChangeInput = jest.fn();
+  const handleClickAddTask = jest.fn();
+  const value = '';
+
+  const renderInput = () => render(
+    <Input
+      value={value}
+      onChange={handleChangeInput}
+      onClick={handleClickAddTask}
+    />,
+  );
```

Comment:
```suggestion
  const renderInput = () => render((
    <Input
      value={value}
      onChange={handleChangeInput}
      onClick={handleClickAddTask}
    />
  ));
```

이렇게 작성하시면 `,`를 없앨 수 있어요.

Reply:
오.. 콤마가 자꾸 붙어서 신경쓰였는데 감사합니다 !